docs/publishers/config.rst

```diff
@@ -385,6 +385,37 @@ loads.
   where annotations are located on the page relative to the current scroll
   position.
 
+.. option:: focus
+
+  .. warning::
+
+      This is an experimental API and may change in future.
```

Comment:
Add _the_ before _future_ here.

Reply:
Copied from above, looks like they are both different than other uses too, gonna change em both to match the rest of the doc